CHANGELOG.md

```diff
@@ -2,4 +2,13 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [0.1.0](https://github.com/keptn-sandbox/keptn-gitea-provisioner-service/compare/0.0.2...0.1.0) (2022-06-29)
+
+
+### ⚠ BREAKING CHANGES
+
+* **release:** Release version 0.1.0
```

Comment:
I think that we can delete one of the two lines here since they have the exact same content.

Reply:
The CHANGELOG.md file is automatically generated by the Release action, i don't thin we should change it if not really necessary

Reply:
Sure it is a little thing, so we can leave it.